test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/ChainingSearchTests.cs

```diff
@@ -61,7 +58,7 @@ public async Task GivenAChainedSearchExpressionOverASimpleParameter_WhenSearched
 
             Bundle bundle = await Client.SearchAsync(ResourceType.DiagnosticReport, query);
 
-            ValidateBundle(bundle, Fixture.SmithDiagnosticReport, Fixture.TrumanDiagnosticReport);
+            ValidateBundle(bundle, Fixture.SmithSnomedDiagnosticReport, Fixture.SmithLoincDiagnosticReport, Fixture.TrumanSnomedDiagnosticReport, Fixture.TrumanLoincDiagnosticReport);
         }
 
         [Fact]
```

Comment:
Can we also add a test to do multiple chaining? e.g., `result.patient.organization.address-city=Seattle`

Reply:
This was covered in one level, but I added another level like you mentioned.